src/Logary/Constants.fs

```diff
@@ -35,6 +35,14 @@ module KnownLiterals =
   [<Literal>]
   let SinkTargetsContextName =  LogaryPrefix +  "sink.targets"
 
+  [<Literal>]
+  let ScopeContextName = LogaryPrefix + "scope"
+
+  [<Literal>]
+  let SpanIdContextName = LogaryPrefix + "spanId"
+  [<Literal>]
+  let SpanInfoContextName = LogaryPrefix + "spanInfo"
```

Comment:
Have you considered the effect on this for the "getOthers" readers and the stuff I added to MessageModule.Patterns?

Reply:
do you think we need to add these two literals ? or we just treat them as other normal user defined context value ?